specification/search/data-plane/Azure.Search/stable/2023-11-01/searchservice.json

```diff
@@ -0,0 +1,10740 @@
+          }
+        ]
+      },
+      "default": "none",
+      "description": "Determines how to process embedded images and image files in Azure blob storage.  Setting the \"imageAction\" configuration to any value other than \"none\" requires that a skillset also be attached to that indexer."
+    },
+    "PdfTextRotationAlgorithm": {
+      "type": "string",
+      "enum": [
+        "none",
+        "detectAngles"
+      ],
+      "x-ms-enum": {
+        "name": "BlobIndexerPDFTextRotationAlgorithm",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "none",
+            "name": "None",
+            "description": "Leverages normal text extraction.  This is the default."
+          },
+          {
+            "value": "detectAngles",
+            "name": "DetectAngles",
+            "description": "May produce better and more readable text extraction from PDF files that have rotated text within them.  Note that there may be a small performance speed impact when this parameter is used.  This parameter only applies to PDF files, and only to PDFs with embedded text.  If the rotated text appears within an embedded image in the PDF, this parameter does not apply."
+          }
+        ]
+      },
+      "default": "none",
+      "description": "Determines algorithm for text extraction from PDF files in Azure blob storage."
+    },
+    "ExecutionEnvironment": {
+      "type": "string",
+      "enum": [
+        "standard",
+        "private"
+      ],
+      "x-ms-enum": {
+        "name": "IndexerExecutionEnvironment",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "standard",
+            "name": "standard",
+            "description": "Indicates that Azure Cognitive Search can determine where the indexer should execute. This is the default environment when nothing is specified and is the recommended value."
+          },
+          {
+            "value": "private",
+            "name": "private",
+            "description": "Indicates that the indexer should run with the environment provisioned specifically for the search service. This should only be specified as the execution environment if the indexer needs to access resources securely over shared private link resources."
+          }
+        ]
+      },
+      "default": "standard",
+      "description": "Specifies the environment in which the indexer should execute."
+    },
+    "FieldMappingFunction": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the field mapping function."
+        },
+        "parameters": {
+          "type": "object",
+          "additionalProperties": true,
```

Comment:
was this intentionally changed? 

![image](https://github.com/Azure/azure-rest-api-specs/assets/68608273/b48653fa-a3ec-468c-b34b-86de23704bea)

Reply:
Yes, this was done to match the previous preview version and the examples.